From 7b7510425cce55e79da66111e3857bb766066f4f Mon Sep 17 00:00:00 2001
From: Piotr Maziarz <piotrx.maziarz@linux.intel.com>
Date: Fri, 10 Jul 2020 11:43:53 +0200
Subject: [PATCH 1198/1214] ASoC: Intel: Skylake: Virtio: Adjust to new IPC
 framework

IPC framework no longer uses lists therefore virtio needed to process
msg in a new way.

Tracked-On: PKT-3636
Change-Id: I54dc571fe11014a63c4071293c4a90285482afe7
---
 sound/soc/intel/skylake/virtio/skl-virtio-fe.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/intel/skylake/virtio/skl-virtio-fe.c b/sound/soc/intel/skylake/virtio/skl-virtio-fe.c
index 20a37d4..d73859b 100644
--- a/sound/soc/intel/skylake/virtio/skl-virtio-fe.c
+++ b/sound/soc/intel/skylake/virtio/skl-virtio-fe.c
@@ -299,8 +299,6 @@ static int vfe_put_inbox_buffer(struct snd_skl_vfe *vfe,
 static int vfe_send_dsp_ipc_msg(struct snd_skl_vfe *vfe,
 	struct ipc_message *msg)
 {
-	msg->complete = true;
-	list_del(&msg->list);
 	sst_ipc_tx_msg_reply_complete(&vfe->sdev.skl_sst->ipc,
 		msg);
 
-- 
2.7.4

